apps/website/src/server/routers/jobs.ts

```diff
@@ -0,0 +1,18 @@
+import { jobPostingTable } from '@bluedot/db';
+import { publicProcedure, router } from '../trpc';
+import db from '../../lib/api/db';
+
+export const jobsRouter = router({
+  getJobs: publicProcedure
+    .query(async () => {
```

Comment:
@marn-in-prod should I add an explicit `.output(z.array(jobSchema))` for output validation?